tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -1555,8 +1627,9 @@ def test_post_bulk_flags_rows_with_errors(client, notify_db, notify_db_session,
 @pytest.mark.parametrize("data_type", ["rows", "csv"])
 @pytest.mark.parametrize("is_scheduled", [True, False])
 @pytest.mark.parametrize("use_sender_id", [True, False])
```

Comment:
Was declared already, but not actually used in the test case previously. That was the mistake, otherwise we would have seen the bug earlier.